example/flake.nix

```diff
@@ -0,0 +1,82 @@
+{
+  description = "Example clojure project";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs";
+    cloure-nix-locker.url = "github:bevuta/clojure-nix-locker";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, clojure-nix-locker, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+        my-clojure-nix-locker = clojure-nix-locker.lib.customLocker {
+          inherit pkgs;
+          # XXX: For some reason this command doesn't work. There is some
+          #      clojure maven download that happens later in the build process.
+          # command = "${pkgs.clojure}/bin/clojure A:build -P";
```

Comment:
Yes, I've witnessed this kind of problem myself too. Sadly there is no reliable way to fetch all deps that a certain tools.build task would need short of running the task itself. I think this is just how things are.